MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```
@@ -32,6 +34,14 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 				self?.refreshControl?.endRefreshing()
 			}
 		}
+		
+		viewModel?.onErrorStateChange = { [weak self] errorText in
+			guard let errorText = errorText else {
+				self?.errorView?.hideMessage()
+				return
+			}
+			self?.errorView?.show(message: errorText)
```

The `guard` should be used to exit early when conditions don't meet. 

Using an `if-else` statement is more readable in this context. 

"If error, show error view. Else, hide error view."